pkg/network/ebpf/c/prebuilt/tracer.c

```diff
@@ -123,24 +124,21 @@ int kprobe__tcp_cleanup_rbuf(struct pt_regs *ctx) {
 }
 
 SEC("kprobe/tcp_close")
-int kprobe__tcp_close(struct pt_regs *ctx) {
-    struct sock *sk;
+int kprobe__tcp_close(struct pt_regs* ctx) {
+    struct sock* sk;
     conn_tuple_t t = {};
     u64 pid_tgid = bpf_get_current_pid_tgid();
     sk = (struct sock *)PT_REGS_PARM1(ctx);
 
-    // Should actually delete something only if the connection never got established
-    bpf_map_delete_elem(&tcp_ongoing_connect_pid, &sk);
```

Comment:
I see `clear_sockfd_maps` added on 140, but I noticed `bpf_map_delete_elem` isn't there. Just curious why `bpf_map_delete_elem` doesn't seem to be needed?